.github/workflows/lint-website.yml

```
@@ -0,0 +1,25 @@
+name: Lint Website
+
+on:
+  push:
+    paths:
+      - ".github/workflows/lint.yml"
+      - "website/**/*.md"
+  pull_request:
+    paths:
+      - ".github/workflows/lint.yml"
```

Leftover from the original file, should probably be

```suggestion
      - ".github/workflows/lint-website.yml"
      - "website/**/*.md"
  pull_request:
    paths:
      - ".github/workflows/lint-website.yml"
```

instead?

---

Yes, good catch!